kpatch/kpatch

```diff
@@ -575,10 +575,14 @@ case "$1" in
 	fi
 
 	[[ ! -e "$MODULE" ]] && die "$PATCH is not installed for kernel $KVER"
-	
 
 	echo "uninstalling $PATCH ($KVER)"
 	rm -f "$MODULE" || die "failed to uninstall module $PATCH"
+
+	if [ -z "$(ls -A "$INSTALLDIR/$KVER")" ]; then
```

Comment:
you could just use
`rmdir --ignore-fail-on-non-empty "$INSTALLDIR/$KVER"`

I should only remove the directory when it's empty.

Reply:
Oh nice, didn't know this existed. This will make it much cleaner.